src/modules/claims/claims.service.ts

```diff
@@ -599,15 +632,15 @@ export class ClaimsService {
       const [issuedToken, vp] = await Promise.all([
         this._didRegistry.issuePublicClaim({
           publicClaim,
-          expirationTimestamp,
+          expirationTimestamp: claimExpirationTimestamp,
         }),
         this.issueVerifiablePresentation({
           subject,
           namespace: claim.claimType,
           version: claim.claimTypeVersion.toString(),
           issuerFields: claim.issuerFields,
           credentialStatus,
-          expirationTimestamp,
+          expirationTimestamp: claimExpirationTimestamp,
```

Comment:
Similar to other comment, we can be sure that now (and in the future) `issueVerifiablePresentation` won't add an `expiry` prop to the VC if this `expirationTimestamp` is `undefined`?

Reply:
I am sure that it won't add this props unless it has some value other the undefined and null or even 0